### DISCUSSION TITLE
[Bug]: Incorrect aspect ration for thumbnails from wide angle cameras when Play Alert Videos is off

### DISCUSSION THREAD

**Discussion by kurtzettel** [blakeblackshear/frigate#13704] [Report a Bug] [answered]
### Checklist

- [X] I have updated to the latest available Frigate version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [ ] I have tried reproducing the issue in [incognito mode](https://www.computerworld.com/article/1719851/how-to-go-incognito-in-chrome-firefox-safari-and-edge.html) to rule out problems with any third party extensions or plugins I have installed.

### Describe the problem you are having

I have a widescreen camera.  The snapshots it takes have a normal ratio.  When they are displayed at the top of the "Live" view the thumbnails are stretched out to have the same aspect ration as the camera, it doesn't match the aspect ratio of the snapshot.
<img width="1404" alt="image" src="https://github.com/user-attachments/assets/2e4842f1-0eeb-43ef-867e-f5f898f0079d">

The thumbnail, if downloaded is 318x180.  The html that renders this shows:
`<div class="relative h-24 4k:h-32" style="aspect-ratio: 3.58209 / 1;" data-state="closed">...`
This ratio is correct for thumbnail videos but not for static thumbnails from an event.

### Steps to reproduce

1. Configure a camera with a wide aspect ratio such as the ANNKE NCD800
2. Change "Play Alert Videos" to false
3. View the live web page when an event occurs.

### Version

0.14.1-f4f3cfa

### In which browser(s) are you experiencing the issue with?

Firefox 130.0

### Frigate config file

```yaml
N/A
```

### docker-compose file or Docker CLI command

```yaml
N/A
```

### Relevant Frigate log output

```shell
N/A
```

### Relevant go2rtc log output

```shell
N/A
```

### Operating system

Other Linux

### Install method

Docker Compose

### Network connection

Wired

### Camera make and model

ANNKE NCD800

### Screenshots of the Frigate UI's System metrics pages

<img width="1367" alt="image" src="https://github.com/user-attachments/assets/c7e6ffc8-658e-4185-a60a-599dfa80a616">

<img width="1368" alt="image" src="https://github.com/user-attachments/assets/57b528a7-79ca-4cce-9479-6b8263018461">

**Comment by NickM-27** [comment, marked as answer]
fixed in https://github.com/blakeblackshear/frigate/pull/13703
